srcpkgs/deadbeef/template

```diff
@@ -1,31 +1,24 @@
 # Template file for 'deadbeef'
 pkgname=deadbeef
-version=1.8.8
+version=1.9.5
 revision=1
 create_wrksrc=yes
 build_style=gnu-configure
-configure_args="--disable-oss --disable-lfm --disable-notify --disable-gtk2"
+configure_args="--disable-oss --disable-notify --disable-gtk2"
 hostmakedepends="automake libtool gettext gettext-devel intltool pkg-config
- yasm clang glib-devel"
+ yasm clang glib-devel libdispatch"
 makedepends="
  alsa-lib-devel dbus-devel faad2-devel ffmpeg-devel gtk+3-devel imlib2-devel
  jansson-devel libcddb-devel libcdio-devel libcurl-devel libflac-devel
  libmad-devel libpng-devel libsamplerate-devel libsndfile-devel libvorbis-devel
- libzip-devel mpg123-devel opusfile-devel pulseaudio-devel wavpack-devel"
-depends="desktop-file-utils hicolor-icon-theme"
+ libzip-devel mpg123-devel opusfile-devel pulseaudio-devel wavpack-devel pipewire-devel git patch clang"
+depends="desktop-file-utils hicolor-icon-theme libdispatch"
```

Comment:
If the package links to this library, it will automatically add it to `depends`. If the library is dynamically loaded, it won't likely need to go into `common/shlibs`.